backuptar/tar.go

```diff
@@ -392,13 +434,9 @@ func WriteBackupStreamFromTarFile(w io.Writer, t *tar.Reader, hdr *tar.Header) (
 			return nil, err
 		}
 	}
+
 	if hdr.Typeflag == tar.TypeSymlink {
```

Comment:
This check seems redundant now that `EncodeReparsePointFromTarHeader` also performs it (albeit in reverse)

Reply:
PTAL.